File: ios/NativeModules/NativeSettingsViewManager.m
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(NativeSettingsViewManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(currency, NSString)
RCT_EXPORT_VIEW_PROPERTY(avatarUrl, NSString)
@end

File: ios/NativeModules/NativeSettingsViewManager.swift
import Foundation
import React

@objc(NativeSettingsViewManager)
class NativeSettingsViewManager: RCTViewManager {
  override func view() -> UIView! {
    return NativeSettingsView()
  }
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
